src/main/java/io/github/moulberry/notenoughupdates/overlays/FarmingSkillOverlay.java

```diff
@@ -94,6 +102,41 @@ private double getCoinsBz(String enchCropName, int numItemsForEnch) {
 		return 0;
 	}
 
+	private void gatherJacobData() {
+		if (System.currentTimeMillis() % 3600000 >= 900000 &&
+			System.currentTimeMillis() % 3600000 <= 2100000) { //Check to see if there is an active Jacob's contest
+			int timeLeftInContest = (20 * 60) - ((int) ((System.currentTimeMillis() % 3600000 - 900000) / 1000));
+			try {
+				Scoreboard scoreboard = Minecraft.getMinecraft().thePlayer.getWorldScoreboard();
```

Comment:
Like nea said, it would be nice to have a Utils method that maybe just returns a `List<String>`